L1Trigger/L1TTrackerDTC/plugins/ProducerFwd.h

```diff
@@ -0,0 +1,39 @@
+#ifndef __L1TTrackerDTC_RODUCERFWD_H__
```

Comment:
it's better to make sure each class only includes the headers that it actually needs, rather than carrying around all possible headers used by the package
(and if you must do the latter, don't put the header file in the plugins directory)

Reply:
I removed the collection of headers